app/src/main/java/org/oppia/app/help/HelpViewModel.kt

```diff
@@ -0,0 +1,11 @@
+package org.oppia.app.help
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.fragment.FragmentScope
+import javax.inject.Inject
+
+/** [ViewModel] for help. */
```

Comment:
Nit:

`/** [ViewModel] for [HelpFragment]. */ `

Reply:
resolved